src/fluent/stdout.h

```diff
@@ -46,7 +46,7 @@ class Stdout {
   }
 
  private:
-  std::set<std::tuple<std::string>> deferred_merge_;
+  std::vector<std::tuple<std::string>> deferred_merge_;
```

Comment:
Simply getting rid of ```make_move_iterator``` at line 46 solves the problem, and it seems cleaner than changing a bunch of data structures and arguments from set to vector...Which change do you prefer?